core/src/main/scala/org/http4s/Message.scala

```diff
@@ -260,6 +262,24 @@ sealed abstract case class Request[F[_]](
   private lazy val connectionInfo = attributes.get(Keys.ConnectionInfo)
 
   lazy val remote: Option[InetSocketAddress] = connectionInfo.map(_.remote)
+
+  /**
+    Convenience method to grab the first X-Forward-For.  This is a convience method.  If
+    all the X-Forwarded-For are needed then you will need to pull it from the headers.
+    */
+  lazy val forwardFor: Option[InetAddress] = headers
```

Comment:
Shouldn't this be `forwardedFor`? To be consistent with the header name.

Reply:
@aeons yes, you are correct.  I will change that.